packages/target-electron/static/webxdc-preload.js

```diff
@@ -138,12 +138,14 @@ class RealtimeListener {
       )
       return Promise.resolve([])
     },
-    sendUpdate: (update, description) =>
-      ipcRenderer.invoke(
-        'webxdc.sendUpdate',
-        JSON.stringify(update),
-        description
-      ),
+    sendUpdate: (update, description) => {
+      if (description) {
+        console.error(
+          'parameter description in sendUpdate is deprecated and will be removed in the future'
+        )
+      }
+      ipcRenderer.invoke('webxdc.sendUpdate', JSON.stringify(update))
```

Comment:
Don't we want to return this promise?

Reply:
This was not changed in this PR

Reply:
Umm, I don't think so.

```javascript
(update, description) =>
  ipcRenderer.invoke(
    'webxdc.sendUpdate',
    JSON.stringify(update),
    description
  )
```

returns a promise.

Reply:
It was changed in this pr, but the type in types definition is void, so in that sense it is correct.

You should normally expect to wait for it to be sent out, because you get you own updates in the listener.
But internally it is async and if we don't await on it, possibly errors will always be uncaught errors, currently.

This might be a specification issue in error handling then: if we say that sendUpdate can fail in core and it should report errors.